transaction-api/src/main/java/com/example/auction/transaction/api/TransactionEvent.java

```diff
@@ -1,11 +1,13 @@
 package com.example.auction.transaction.api;
 
 import java.util.UUID;
+import lombok.Value;
 
 public abstract class TransactionEvent {
 
```

Comment:
All the code in `transaction-xxx` is WIP and it'd be good to not add more complexity on it until we have a clear idea what is it we want to do with it.